src/PowerLimiterSolarInverter.cpp

```diff
@@ -82,14 +82,23 @@ uint16_t PowerLimiterSolarInverter::getMaxIncreaseWatts() const
         return maxTotalIncrease;
     }
 
-    int16_t maxPowerPerMppt = getConfiguredMaxPowerWatts() / dcTotalMppts;
+    // for inverter with PDL we use the max power of the inverter because each MPPT can deliver its max power,
+    // for inverters without PDL we use the configured max power, because the limit will be divided equally across the MPPTs by the inverter.
+    int16_t inverterMaxPower = _spInverter->supportsPowerDistributionLogic() ? getInverterMaxPowerWatts() : getConfiguredMaxPowerWatts();
```

Comment:
The above comment about PDL vs overscaling could be solved as shown below

```suggestion
    // for inverters without PDL we use the configured max power, because the limit will be divided equally across the MPPTs by the inverter.
    int16_t inverterMaxPower = getConfiguredMaxPowerWatts();

    // for inverter with PDL or when overscaling is enabled we use the max power of the inverter because each MPPT can deliver its max power.
    if (_spInverter->supportsPowerDistributionLogic() || _config.UseOverscaling) {
        inverterMaxPower = getInverterMaxPowerWatts();
    }
```